packages/dds/cell/src/test/cell.spec.ts

```diff
@@ -24,7 +24,10 @@ function createConnectedCell(
 ): ISharedCell {
 	// Create and connect a second SharedCell.
 	const dataStoreRuntime = new MockFluidDataStoreRuntime();
-	dataStoreRuntime.options = options ?? dataStoreRuntime.options;
+	// TODO this option shouldn't live here - this options object is global to the container
```

Comment:
i think the first thing is to change the type of  dataStoreRuntime be something other than ILoaderOptions, probably Record<string | number, any> to start. that would also get rid of all the casting this change introduces

Reply:
i'm also not positive that the province of this object actually comes all the way from the loader. the type might just be a lie

Reply:
I left this annoying intentionally - my thought here was to discourage folks from writing new values to options (with this PR as-is, they'll get an error).  This is with the goal of deterring new (possibly dangerous) usage.

I think the likely end state being that the dataStoreRuntime.options member goes away entirely.

Reply:
> i'm also not positive that the province of this object actually comes all the way from the loader. the type might just be a lie

The loader options get shallow-copied per-Container - so shallow modifications are global to the Container, but deep modifications of true loader options (e.g. individual IClient properties) are global to the Loader.  But yeah they get plumbed all the way through.

Reply:
I'm looking at an option to split the typing (maybe this is what you originally meant with your comment) - ILoaderOptions with only known types for the Loader constructor param, but everywhere else gets JUST the record-like.  Which then would point to a followup change that just severs the connection at the loader/runtime boundary, with the runtime creating an empty object as its starting options and getting nothing from the loader.

Reply:
Updated in latest!